task/06-conditions-n-loops-tasks.js

```diff
@@ -296,7 +343,24 @@ function getDigitalRoot(num) {
  *   '{[(<{[]}>)]}' = true 
  */
 function isBracketsBalanced(str) {
-    throw new Error('Not implemented');
+    let arr = [];
+    let open = ['[', '{', '(', '<'];
+    let close = [']', '}', ')', '>'];
+
+    if (str.length % 2 !== 0) {
+        return false;
+    }
+    for (let i = 0; i < str.length; i++) {
+        if (open.indexOf(str[i]) >= 0) {
+            arr.push(str[i]);
+        }
+        if (close.indexOf(str[i]) >= 0) {
+            if (close.indexOf(str[i]) == open.indexOf(arr[arr.length - 1])) {
```

Comment:
"operator === is always preferable
sometimes ''==' may cause an error
(check at console :
'0' == 0
and '0' === 0)"